tests/create_test_data.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
this should live in `tests/notebooks/` (might require updates to some of the paths in the notebook)